### PR TITLE
adding DappMaintenance logic back

### PR DESCRIPTION
**Pull request by clementbalestrat**
Quick fix to bring DappMaintenance logic back.

### CODE REVIEW

File: src/pages/Home/NewUserPromo/NewUserPromo.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ import { HeadingMedium, BodyLarge } from 'src/components/Typography';
 import { dataLargeCSS } from 'src/components/Typography/Data';
 import Link from 'src/components/Link';
 
-import { ReactComponent as BorrowsETHIcon } from 'src/assets/images/splash/borrow-sETH.svg';
-import { ReactComponent as BuysETHIcon } from 'src/assets/images/splash/buy-sETH.svg';
-import { ReactComponent as BuysUSDIcon } from 'src/assets/images/splash/buy-sUSD.svg';
+import { ReactComponent as BorrowsETHIcon } from 'src/assets/images/splash/borrow-seth.svg';
+import { ReactComponent as BuysETHIcon } from 'src/assets/images/splash/buy-seth.svg';
+import { ReactComponent as BuysUSDIcon } from 'src/assets/images/splash/buy-susd.svg';
```
Comment (evgenyboxer) [inline src/pages/Home/NewUserPromo/NewUserPromo.js]: why is this needed?

Comment (clementbalestrat) [reply src/pages/Home/NewUserPromo/NewUserPromo.js]: It breaks my linter. My file system says they're lowercase. Isn't it the same for you?
